stonks/__init__.py

```diff
@@ -0,0 +1,64 @@
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+from webdriver_manager.firefox import GeckoDriverManager
+from bs4 import BeautifulSoup
+import time
+import re
+from .types import StonksResult
+
+
+class Handler:
+    """
```

Comment:
The first line needs to be a resume about what this class do, check in this link
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings